src/main/java/Duke.java

```diff
@@ -1,10 +1,268 @@
+                    }else {
+                        int number = firstSlashTracker(order);
+                        Task item = new Task(2, 0, order.substring(5, number) + "(at: " +
+                                order.substring(number + 3) + ")");
+                        orders.add(item);
+                    }
+
+                }else {
+                    Task item = new Task(4, 0, order);
+                    orders.add(item);
+                }
+
+            }
+            order = sc.nextLine();
+        }
+        System.out.println("Ka To:");
+        System.out.println(" ");
+        System.out.println("I am happy to serve you. See you soon!");
+    }
+
     public static void main(String[] args) {
+        String filePath = "./data/duke.txt";
+
+        try {
+            File myFile = new File(filePath);
+            Scanner myReader = new Scanner(myFile);
+            while (myReader.hasNextLine()) {
+                String data = myReader.nextLine();
+                System.out.println(data);
+            }
+            myReader.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Creating the file to save");
+            try {
+                String folderPath = "./data";
+                Path path = Paths.get(folderPath);
+
+                Files.createDirectories(path);
+
+                System.out.println("Directory is created!");
+
+                //Files.createDirectory(path);
+
+            } catch (IOException error) {
+                System.err.println("Failed to create directory!" + error.getMessage());
+            }
+            try {
+                File fileCreator = new File(filePath);
+                if (fileCreator.createNewFile()) {
+                    System.out.println("File created: " + fileCreator.getName());
+                } else {
+                    System.out.println("File already exists.");
+                }
+            } catch (IOException ex) {
+                System.out.println("An error occurred.");
+                e.printStackTrace();
+            }
+        }
+        try {
+            File myFile = new File(filePath);
+            Scanner myReader = new Scanner(myFile);
+            while (myReader.hasNextLine()) {
+                String data = myReader.nextLine();
+                System.out.println(data);
+            }
+            myReader.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Creating the file to save");
+        }
+
+        try {
+            FileWriter myWriter = new FileWriter(filePath);
+            //myWriter.write("Files in Java might be tricky, but it is fun enough!");
+            myWriter.close();
+            System.out.println("All changes successfully saved to the storage!");
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
```

Comment:
I'm sorry but I don't really understand this part. Would it be possible to combine all the try blocks together into one and have multiple catch blocks for different errors instead?